CONTRIBUTING.md

```diff
@@ -1,5 +1,3 @@
-
-
 ---
```

Comment:
Removing the first two lines of the document messes up the formatting of the first section.

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix